### ISSUE TITLE
Align multi-agent orchestrator/grounder phrasing with the pilot's native distribution (snap-to-native)

### ISSUE THREAD

**Issue by SoyGema** [lovellai-dev/odyssey#73]
## Context

The multi-agent eval arms feed the PILOT (OpenVLA) instruction text authored by the SPECIALIST/orchestrator, not LIBERO's own native instruction:

- **planner arm** (`coordination: planning`, PR #64) — decomposed sub-steps.
- **delegation arm** (`coordination: delegation`, PR #68) — `pick up {grounded}` phrases.
- **orchestration arm** (`coordination: orchestration`, regime D — in design) — router-emitted sub-tasks.

`openvla-7b-finetuned-libero-object` was fine-tuned on LIBERO's *own* instructions, so every non-native phrasing is out-of-distribution for it. This can depress **all** multi-agent arms vs the single-agent baseline **regardless of coordination quality** — i.e. it confounds the planner-vs-delegation-vs-orchestration comparison (a good orchestrator can still lose because the pilot never saw that phrasing). See the threat-to-validity note in the internal experiment doc (§4.6).

## Problem

We need a way to keep the orchestrator/grounder's reasoning freedom while ensuring what actually reaches `pilot.act(image, instruction)` stays inside the pilot's training distribution.

## Proposed approach — `InstructionAdapter` (snap-to-native), opt-in

An optional adapter, `config.align_phrasing` (default **off**), that projects the emitted phrase to the nearest LIBERO-native instruction before it reaches the pilot, e.g.:

```
"pick up the red mug on the left"  ->  "<nearest LIBERO-native instruction for this task>"
```

- Reusable by **all three** multi-agent arms (planning / delegation / orchestration), not only orchestration.
- Zero extra VRAM if the matching is lexical/embedding-based over the task's own `language`.

## Open design questions (need a decision)

1. **Similarity metric** — how is "nearest" computed: sentence embeddings, lexical overlap, or a fixed per-task map derived from LIBERO's instruction templates?
2. **The core tension** — snapping to the native instruction risks *erasing the very signal* the orchestrator/grounder adds (scene-specific nuance). Where is the balance between "align to avoid pilot degradation" and "let the orchestrator express nuance"? Is a controlled *vocabulary* (constrain tokens, not whole-sentence snap) a better middle ground?
3. **Scope + default** — apply to all multi-agent arms or orchestration only? Default on or off? (Leaning: shared, default off, opt-in per mission.)
4. **Measurement** — quantify the OOD gap with/without the adapter against the single-agent baseline, so we can show the adapter recovers success rate rather than just assuming it.

## Alternative / complementary track (out of scope here)

**Invocation-aligned pilot fine-tuning** — the rigorous fix: fine-tune the pilot on the phrasing distribution the orchestrator actually emits (the VLAs-as-Tools warning: wrapping a stock VLA underperforms unless the executor is post-trained for the invocation interface). Separate training track; tracked here only as the long-term counterpart to the cheap in-loop adapter.

## Notes

- A generalist pilot (e.g. π0.5, GR00T) is far less sensitive to non-native phrasing than the narrowly fine-tuned OpenVLA-libero, so this issue is most acute specifically for the OpenVLA-libero pilot and shrinks as we move to generalist pilots.

## Acceptance (proposed)

- [ ] Decide metric + vocabulary-vs-sentence-snap (Q1/Q2).
- [ ] `InstructionAdapter` behind `config.align_phrasing` (default off), shared across arms.
- [ ] A/B measurement of success rate vs single-agent baseline with the adapter on/off.
- [ ] Docs: update the experiment doc + `examples/franka-libero/README.md`.

Related: PR #64 (planner + semantic transitions), PR #68 (delegation arm), the orchestration (regime D) arm in design.
